imod/mf6/package.py

```diff
@@ -40,8 +40,9 @@ class Package(PackageBase, abc.ABC):
     """
 
     _pkg_id = ""
-    _init_schemata = {}
```

Comment:
I don't think ``ValueSchema`` is the right type here, as that is quite a specific implementation. I think we need to define a ``SchemaType`` or something for this.

Reply:
I means defining something like this in ``schemata.py``

```python
from typing import TypeVar

SchemaType = TypeVar("SchemaType", bound="BaseSchema")
```

Reply:
Applied this